packages/blocks/shuffle/src/components/item.tsx

```diff
@@ -40,44 +42,36 @@ export const Item = forwardRef<HTMLLIElement, ItemProps>(
       listeners,
       dragOverlay,
       readonly,
+      isEntity,
     },
     ref,
   ) => {
     const [isHovered, setIsHovered] = useState(false);
 
     return (
-      <ListItem
+      <SListItem
         ref={ref}
         onMouseOver={() => setIsHovered(true)}
         onMouseOut={() => setIsHovered(false)}
         disablePadding
-        sx={{
-          ...style,
-          marginBottom: 2,
-          opacity: isDragging ? 0 : 1,
-          outlineColor: ({ palette }) => palette.primary.light,
-        }}
+        sx={{ ...style, opacity: isDragging ? 0 : 1 }}
         {...attributes}
       >
-        <Paper
-          sx={{
-            display: "flex",
-            width: 1,
-            paddingX: 2,
-            paddingY: 1,
-            background: ({ palette }) => palette.grey[50],
-            ...paperStyle,
-          }}
-          ref={dragOverlay}
-        >
+        <SPaper sx={paperStyle} ref={dragOverlay}>
+          <Tooltip title={isEntity ? "This item is linked to an entity" : ""}>
+            <SLinkIconWrapper>
+              {isEntity && <LinkIcon fontSize="small" />}
+            </SLinkIconWrapper>
+          </Tooltip>
+
           <TextField
             multiline
             fullWidth
             variant="standard"
             sx={{
               border: "none",
               outline: "none",
-              caretColor: readonly ? "transparent" : "initial",
+              caretColor: readonly || isEntity ? "transparent" : "initial",
```

Comment:
@yusufkinatas did you see this one?

Reply:
Updated it with last commits, now it should be cleaner. Did not used a "mode",  since I need "readonly" and "linkedToEntity" seperately to decide how to render the component.